### DISCUSSION TITLE
Changing skybox or environment rotation on scroll

### DISCUSSION THREAD

**Discussion by trg7** [google/model-viewer#2009] [Q&A]
I'm currently working on a project where I have an architectural model and would like to build interactivity into the scroll where the building rotates slowly and as it rotates the skybox/environment light rotates faster and creates an almost time lapse affect where the sun spins around the model.

From your examples provided I've managed to get the rotate on scroll interactivity in with some adjustments to this code https://modelviewer.dev/examples/stagingandcameras/#orbitAndScroll

And have managed to get the skybox rotating when holding shift and mouse drag. https://modelviewer.dev/examples/stagingandcameras/#turnSkybox

I was wondering if at all it would be possible to use the same principle as above example (turnSkybox) but convert to rotating the skybox/environment image on scroll? If anyone knew if/how this possible it would be super helpful! 

(apologies if this somewhat basic question I'm new to coding)

Many Thanks!

**Comment by elalish** [comment]
We don't have that ability with our declarative markup like we have for the orbit. I would recommend using a scroll event listener to update both the `turntableRotation` and the `cameraOrbit` instead.
